RxSwiftWorkshopTests/Utilities/Equatability.swift

```diff
@@ -0,0 +1,13 @@
+//
+// Created by Krzysztof Siejkowski on 03/05/2017.
+// Copyright (c) 2017 Mobile Academy. All rights reserved.
+//
+
+import Foundation
+import CoreLocation
+
+extension CLLocationCoordinate2D: Equatable {
```

Comment:
I've switched to file per concept and only extact the separate files when there's too much of them to read in a single file